app-backend/backsplash-core/src/main/scala/com/rasterfoundry/backsplash/BacksplashImage.scala

```diff
@@ -247,10 +247,7 @@ sealed trait BacksplashImage[F[_]] extends LazyLogging {
 }
 
 object BacksplashImage {
-  val tmsLevels = {
-    val scheme = ZoomedLayoutScheme(WebMercator, 256)
-    for (zoom <- 0 to 64) yield scheme.levelForZoom(zoom).layout
-  }.toArray
+  val tmsLevels = tiling.tmsLevels
```

Comment:
This value needed to move up into `db` because it's needed to figure out the empty tile for combining vector geometries. Rather than introduce a bunch of noise around switching to a `db` subproject object in the `backsplash-core` info, I opted to keep the value here but grab it from the upstream value. I can be talked out of this if it looks too ugly.